#
# request 
#
bool personality_request
---
#
#
int8 personality_profile

